themes/minimal/minimal.ps1

```diff
@@ -0,0 +1,15 @@
+# add title escape time, jobs, load and battery
+PS1="${LP_PS1_PREFIX}"
+# add user, host, permissions colon, working directory, and dirstack
+PS1+="${LP_BRACKET_OPEN}${LP_USER}${LP_HOST}${LP_PERM}${LP_PWD}${LP_BRACKET_CLOSE}"
+
+# Add VCS infos
+# If root, the info has not been collected unless LP_ENABLE_VCS_ROOT
+# is set.
+PS1+="${LP_VCS}"
+
+# add return code and prompt mark
```

Comment:
Most of these comments are incorrect:
> add title escape time, jobs, load and battery

None of that is added.
> dirstack

That isn't in there either.
> add return code and prompt mark

Runtime is also added here.

Reply:
Fixed the related comments in `liquidprompt` as well.